### DISCUSSION TITLE
Is it possible to add annotations to lightweight-charts?

### DISCUSSION THREAD

**Discussion by fx1357** [tradingview/lightweight-charts#1084] [Q&A]
I would like to add annotations(you could say it's a text note) to the tradingview as shown in the following picture.

![pp](https://user-images.githubusercontent.com/33419328/168990728-d4d90a3e-7769-4c09-9f8d-94f8c9890515.png)

I tried to use series marker. but, it bothers me that I can't add & design a box that wraps around text.

And I don't want to use Technical Analysis Charts because I only need that function(annotations). I don't even have a completed homepage to fill out the application form.

Is there a way to add annotations to the tradingview(lightweight-charts)?

**Comment by romfrancois** [comment]
Hi @fx1357 !

You could use [markers](https://tradingview.github.io/lightweight-charts/docs/api/interfaces/ISeriesApi#setmarkers) and have something like:

```
const timer = new Date().getTime();
const markers = [
  {
        time: timer,
        position: 'belowBar',
        color: 'orange',
        shape: 'arrowUp',
        id: 'id4',
        text: 'example',
        size: 2,
    },
];
yourSeries.setMarkers(markers);
```

**Reply by fx1357** [reply]
But.... I want to make a textbox(a rectangle) for text and customize design of the box.

**Comment by romfrancois** [comment]
That's not possible at the moment.

**Reply by wwarodom** [reply]
It will be very useful to have this function in a TV Chart.

**Reply by fx1357** [reply]
> That's not possible at the moment.

Thank you very much for your answer@!!




> It will be very useful to have this function in a TV Chart.

I'm desperate for that function, too
